README.md

```diff
@@ -24,6 +25,8 @@ The use case is a streaming pipeline built around live edits to real Wikipedia p
 
 ![image](images/drawing.png)
 
+In this demo we have chosen to enable SSL for encryption and authentication for Kafka and SASL for Kafka communications with ZooKeeper. This provides example of how a production cluster would be configured in a secured environment.
```

Comment:
"a production cluster" --> "components in the Confluent Platform" (to avoid people thinking that this Docker setup is for production)